apps/condo/domains/resident/access/ServiceConsumer.js

```diff
@@ -0,0 +1,35 @@
+
+const { ServiceConsumer, Resident } = require('@condo/domains/resident/utils/serverSchema')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+async function canReadServiceConsumers ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return {}
+    if (user.type === RESIDENT) {
+        return {
+            resident: { user: { id: user.id } },
+        }
+    }
+    return false
+}
+
+async function canManageServiceConsumers ({ authentication: { item: user }, context, originalInput, operation, itemId }) {
+    if (!user) return false
+    if (user.isAdmin) return true
+    if (user.type === RESIDENT) {
+        if (operation === 'update') {
```

Comment:
you can use it to change the account number to invalid!

Reply:
fixed